.mise.toml

```
@@ -81,10 +91,47 @@ run = """
 #!/bin/bash
 cp .env.example .env
 source .env
-poetry run pytest . --ignore ./tilt
+poetry run pytest . --ignore ./tilt $@
 """
 
 [tasks."tests:unit"]
 description = "Run unit tests"
 depends = ["poetry:install"]
-run = "poetry run pytest app --ignore=app/tests/e2e"
+run = "poetry run pytest app --ignore=app/tests/e2e $@"
+
+[tasks."tests:e2e"]
+description = "Run e2e tests"
+depends = ["poetry:install"]
+run = """
+#!/bin/bash
+cp .env.example .env
+source .env
+# pytest app/tests/e2e
+poetry run pytest --dist loadfile app/tests/e2e $@
+"""
+
+[tasks."fmt:check"]
+description = "Check all files for formatting"
+run = """
+#!/bin/bash
+isort . --check --profile black --diff
+black . --check --diff
+"""
+
+[tasks."fmt:check:tiltfiles"]
+description = "Check Tiltfiles for formatting"
+run = """
+#!/bin/bash
+find . -type f -name "Tiltfile" | while read -r file; do
+    black --check --diff "$file"
+  done
+"""
+
+[tasks."fmt:tiltfiles"]
+description = "Format Tiltfiles"
+run = """
+#!/bin/bash
+find . -type f -name "Tiltfile" | while read -r file; do
+    black "$file"
+  done
+"""
```

I think it'd be better to merge the format tasks:

```suggestion
[tasks.fmt]
description = "Format files"
run = """
#!/bin/bash
isort . --profile black --diff
black . --diff
find . -type f -name "Tiltfile" | while read -r file; do
    black "$file"
  done
"""

[tasks."fmt:check"]
description = "Check all files for formatting"
run = """
#!/bin/bash
isort . --check --profile black --diff
black . --check --diff
find . -type f -name "Tiltfile" | while read -r file; do
    black --check --diff "$file"
  done
"""
```